WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1219,127 +1150,12 @@ private void showQuickStartDialogMigration() {
                 "",
                 "",
                 ""
-        );
+                              );
```

Comment:
```suggestion
        );
```

Is looks like automatic formatting went wild with this indention.  Can we revert it back?

Reply:
It's indeed weird, but if I revert it, it's just going to come back at some point as long as       `<option name="ALIGN_MULTILINE_METHOD_BRACKETS" value="true" />` is in our codestyle. I'll move it to the previous line so it will look a bit better.